packages/gamut/src/AppBar/AppBarSection.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import cx from 'classnames';
+import styles from './styles/index.scss';
+
+export type AppBarSectionProps = {
+  position?: 'left' | 'center' | 'right';
+  className?: string;
+};
+
+class AppBarSection extends React.Component<AppBarSectionProps> {
+  render() {
+    const classes = cx({
+      [styles.section]: true,
+      [styles.sectionRight]: this.props.position === 'right',
+      [styles.sectionLeft]: this.props.position === 'left',
+      [styles.sectionCenter]: this.props.position === 'center',
+      [`${this.props.className}`]: !!this.props.className,
+    });
```

Comment:
Fun fact, `classnames`/`cx` does truthiness checking for us ✨ 

```suggestion
    },
    this.props.className);
```